packages/ui/src/PageHeader.tsx

```diff
@@ -97,3 +102,23 @@ PageHeader.Image = function PageHeaderImage({
     </div>
   )
 }
+
+function Description({
+  description,
+}: {
+  description: PageHeaderProps['description']
+}) {
+  if (!description) return null
+
+  if (typeof description === 'object' && 'text' in description) {
+    return (
+      <p
+        className={clsx(description.isClamped && 'line-clamp-3 text-ellipsis')}
+      >
+        {description.text}
+      </p>
+    )
+  }
+
+  return <DescriptionText>{description}</DescriptionText>
+}
```

Comment:
I think it would be cleaner if we add `isClamped` to `DescriptionText`. What do you think?

Reply:
Yes, I like that!